<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated with glade 3.38.2 -->
<interface domain="transmission-gtk">
  <requires lib="gtk+" version="3.24"/>
  <object class="GtkDialog" id="MakeDialog">
    <property name="can-focus">False</property>
    <property name="border-width">6</property>
    <property name="title" translatable="yes">New Torrent</property>
    <property name="type-hint">dialog</property>
    <child internal-child="vbox">
      <object class="GtkBox" id="dialog_layout">
        <property name="can-focus">False</property>
        <property name="orientation">vertical</property>
        <property name="spacing">6</property>
        <child internal-child="action_area">
          <object class="GtkButtonBox" id="dialog_buttons">
            <property name="can-focus">False</property>
            <property name="layout-style">end</property>
            <child>
              <object class="GtkButton" id="close_button">
                <property name="label" translatable="yes">_Close</property>
                <property name="visible">True</property>
                <property name="can-focus">True</property>
                <property name="can-default">True</property>
                <property name="receives-default">True</property>
                <property name="use-underline">True</property>
              </object>
              <packing>
                <property name="expand">True</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="new_button">
                <property name="label" translatable="yes">_New</property>
                <property name="visible">True</property>
                <property name="can-focus">True</property>
                <property name="can-default">True</property>
                <property name="receives-default">True</property>
                <property name="use-underline">True</property>
              </object>
              <packing>
                <property name="expand">True</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">False</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkBox" id="dialog_content_layout">
            <property name="visible">True</property>
            <property name="can-focus">False</property>
            <property name="orientation">vertical</property>
            <property name="spacing">6</property>
            <child>
              <object class="GtkLabel" id="files_section_label">
                <property name="visible">True</property>
                <property name="can-focus">False</property>
                <property name="hexpand">True</property>
                <property name="label" translatable="yes">Files</property>
                <property name="xalign">0</property>
                <attributes>
                  <attribute name="weight" value="bold"/>
                </attributes>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <!-- n-columns=2 n-rows=5 -->
              <object class="GtkGrid" id="files_section_layout">
                <property name="visible">True</property>
                <property name="can-focus">False</property>
                <property name="margin-start">18</property>
                <property name="row-spacing">6</property>
                <property name="column-spacing">12</property>
                <child>
                  <object class="GtkLabel" id="destination_label">
                    <property name="visible">True</property>
                    <property name="can-focus">False</property>
                    <property name="label" translatable="yes">Sa_ve to:</property>
                    <property name="use-underline">True</property>
                    <property name="mnemonic-widget">destination_button</property>
                    <property name="xalign">0</property>
                  </object>
                  <packing>
                    <property name="left-attach">0</property>
                    <property name="top-attach">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkFileChooserButton" id="destination_button">
                    <property name="visible">True</property>
                    <property name="can-focus">False</property>
                    <property name="hexpand">True</property>
                    <property name="action">select-folder</property>
                    <property name="title" translatable="yes"/>
                  </object>
                  <packing>
                    <property name="left-attach">1</property>
                    <property name="top-attach">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkRadioButton" id="source_folder_radio">
                    <property name="label" translatable="yes">Source F_older:</property>
                    <property name="visible">True</property>
                    <property name="can-focus">True</property>
                    <property name="receives-default">False</property>
                    <property name="valign">center</property>
                    <property name="use-underline">True</property>
                    <property name="draw-indicator">True</property>
                  </object>
                  <packing>
                    <property name="left-attach">0</property>
                    <property name="top-attach">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkFileChooserButton" id="source_folder_button">
                    <property name="visible">True</property>
                    <property name="sensitive" bind-source="source_folder_radio" bind-property="active" bind-flags="sync-create">False</property>
                    <property name="can-focus">False</property>
                    <property name="hexpand">True</property>
                    <property name="action">select-folder</property>
                    <property name="title" translatable="yes"/>
                  </object>
                  <packing>
                    <property name="left-attach">1</property>
                    <property name="top-attach">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkRadioButton" id="source_file_radio">
                    <property name="label" translatable="yes">Source _File:</property>
                    <property name="visible">True</property>
                    <property name="can-focus">True</property>
                    <property name="receives-default">False</property>
                    <property name="valign">center</property>
                    <property name="use-underline">True</property>
                    <property name="active">True</property>
                    <property name="draw-indicator">True</property>
                    <property name="group">source_folder_radio</property>
                  </object>
                  <packing>
                    <property name="left-attach">0</property>
                    <property name="top-attach">2</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkFileChooserButton" id="source_file_button">
                    <property name="visible">True</property>
                    <property name="sensitive" bind-source="source_file_radio" bind-property="active" bind-flags="sync-create">False</property>
                    <property name="can-focus">False</property>
                    <property name="hexpand">True</property>
                    <property name="title" translatable="yes"/>
                  </object>
                  <packing>
                    <property name="left-attach">1</property>
                    <property name="top-attach">2</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkLabel" id="source_size_label">
                    <property name="visible">True</property>
                    <property name="can-focus">False</property>
                    <property name="hexpand">True</property>
                    <property name="label" translatable="yes">No source selected</property>
                    <property name="xalign">0</property>
                    <attributes>
                      <attribute name="style" value="italic"/>
                    </attributes>
                  </object>
                  <packing>
                    <property name="left-attach">1</property>
                    <property name="top-attach">3</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkLabel" id="piece_size_label">
                    <property name="visible">True</property>
                    <property name="can-focus">False</property>
                    <property name="label" translatable="yes">Piece size:</property>
                    <property name="use-underline">True</property>
                    <property name="mnemonic-widget">piece_size_scale</property>
                    <property name="xalign">0</property>
                  </object>
                  <packing>
                    <property name="left-attach">0</property>
                    <property name="top-attach">4</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkScale" id="piece_size_scale">
                    <property name="visible">True</property>
                    <property name="can-focus">True</property>
                    <property name="hexpand">True</property>
                    <property name="digits">0</property>
                    <property name="draw-value">False</property>
                    <property name="value-pos">left</property>
                  </object>
                  <packing>
                    <property name="left-attach">1</property>
                    <property name="top-attach">4</property>
                  </packing>
                </child>
                <child>
                  <placeholder/>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkFixed" id="properties_section_spacer">
                <property name="height-request">6</property>
                <property name="visible">True</property>
                <property name="can-focus">False</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">2</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="properties_section_label">
                <property name="visible">True</property>
                <property name="can-focus">False</property>
                <property name="hexpand">True</property>
                <property name="label" translatable="yes">Properties</property>
                <property name="xalign">0</property>
                <attributes>
                  <attribute name="weight" value="bold"/>
                </attributes>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">3</property>
              </packing>
            </child>
            <child>
              <!-- n-columns=2 n-rows=4 -->
              <object class="GtkGrid" id="properties_section_layout">
                <property name="visible">True</property>
                <property name="can-focus">False</property>
                <property name="margin-start">18</property>
                <property name="row-spacing">6</property>
                <property name="column-spacing">12</property>
                <child>
                  <object class="GtkLabel" id="trackers_label">
                    <property name="visible">True</property>
                    <property name="can-focus">False</property>
                    <property name="valign">start</property>
                    <property name="ypad">6</property>
                    <property name="label" translatable="yes">_Trackers:</property>
                    <property name="use-underline">True</property>
                    <property name="mnemonic-widget">trackers_view</property>
                    <property name="xalign">0</property>
                  </object>
                  <packing>
                    <property name="left-attach">0</property>
                    <property name="top-attach">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkBox" id="trackers_layout">
                    <property name="visible">True</property>
                    <property name="can-focus">False</property>
                    <property name="hexpand">True</property>
                    <property name="vexpand">True</property>
                    <property name="orientation">vertical</property>
                    <property name="spacing">3</property>
                    <child>
                      <object class="GtkScrolledWindow" id="trackers_view_scroll">
                        <property name="height-request">80</property>
                        <property name="visible">True</property>
                        <property name="can-focus">True</property>
                        <property name="hexpand">True</property>
                        <property name="vexpand">True</property>
                        <property name="shadow-type">in</property>
                        <child>
                          <object class="GtkTextView" id="trackers_view">
                            <property name="visible">True</property>
                            <property name="can-focus">True</property>
                            <property name="accepts-tab">False</property>
                          </object>
                        </child>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="trackers_description_label">
                        <property name="visible">True</property>
                        <property name="can-focus">False</property>
                        <property name="hexpand">True</property>
                        <property name="label" translatable="yes">To add a backup URL, add it on the next line after a primary URL.
To add a new primary URL, add it after a blank line.</property>
                        <property name="xalign">0</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">1</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="left-attach">1</property>
                    <property name="top-attach">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkCheckButton" id="comment_check">
                    <property name="label" translatable="yes">Co_mment:</property>
                    <property name="visible">True</property>
                    <property name="can-focus">True</property>
                    <property name="receives-default">False</property>
                    <property name="valign">center</property>
                    <property name="use-underline">True</property>
                    <property name="draw-indicator">True</property>
                  </object>
                  <packing>
                    <property name="left-attach">0</property>
                    <property name="top-attach">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkEntry" id="comment_entry">
                    <property name="visible">True</property>
                    <property name="sensitive" bind-source="comment_check" bind-property="active" bind-flags="sync-create">False</property>
                    <property name="can-focus">True</property>
                    <property name="hexpand">True</property>
                  </object>
                  <packing>
                    <property name="left-attach">1</property>
                    <property name="top-attach">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkCheckButton" id="source_check">
                    <property name="label" translatable="yes">_Source:</property>
                    <property name="visible">True</property>
                    <property name="can-focus">True</property>
                    <property name="receives-default">False</property>
                    <property name="valign">center</property>
                    <property name="use-underline">True</property>
                    <property name="draw-indicator">True</property>
                  </object>
                  <packing>
                    <property name="left-attach">0</property>
                    <property name="top-attach">2</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkEntry" id="source_entry">
                    <property name="visible">True</property>
                    <property name="sensitive" bind-source="source_check" bind-property="active" bind-flags="sync-create">False</property>
                    <property name="can-focus">True</property>
                    <property name="hexpand">True</property>
                  </object>
                  <packing>
                    <property name="left-attach">1</property>
                    <property name="top-attach">2</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkCheckButton" id="private_check">
                    <property name="label" translatable="yes">_Private torrent</property>
                    <property name="visible">True</property>
                    <property name="can-focus">True</property>
                    <property name="receives-default">False</property>
                    <property name="valign">center</property>
                    <property name="use-underline">True</property>
                    <property name="draw-indicator">True</property>
                  </object>
                  <packing>
                    <property name="left-attach">0</property>
                    <property name="top-attach">3</property>
                    <property name="width">2</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">4</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">0</property>
          </packing>
        </child>
      </object>
    </child>
    <action-widgets>
      <action-widget response="close">close_button</action-widget>
      <action-widget response="accept">new_button</action-widget>
    </action-widgets>
  </object>
  <object class="GtkSizeGroup" id="labels_width_group">
    <widgets>
      <widget name="destination_label"/>
      <widget name="source_folder_radio"/>
      <widget name="source_file_radio"/>
      <widget name="piece_size_label"/>
      <widget name="trackers_label"/>
      <widget name="comment_check"/>
      <widget name="source_check"/>
    </widgets>
  </object>
</interface>
